scripts/codebase_utils.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+for cmd in bazel; do
+    which "$cmd" >/dev/null
+done
+
```

Comment:
Are we even building anything without that DIRS env var?

Reply:
Which DIR? `//...` means "everything". Since there are no experimental dir here then it is good enough.

Reply:
CRIS_CODE_DIRS, at least set it to empty string to keep interface consistent.

Reply:
That does not work. CRIS_CODE_DIRS will be parsed to bazel targets `//dirname/...`, but for here we need `//...` to build all, and `///...` does not work

Reply:
Current format_all relies on it.

Reply:
Ack